src/main/java/com/znsio/e2e/tools/Driver.java

```diff
@@ -1,12 +1,13 @@
 package com.znsio.e2e.tools;
 
+import com.google.common.collect.ImmutableMap;
 import com.znsio.e2e.runner.Runner;
-import io.appium.java_client.AppiumDriver;
-import io.appium.java_client.MobileBy;
-import io.appium.java_client.TouchAction;
+import io.appium.java_client.*;
```

Comment:
Can you replace with specific imports instead of .*?

Reply:
it did it on its own. updated it in the new push